### PR TITLE
NFDIV-719: Add functional tests for you need to review page

### PR DESCRIPTION
**Pull request by tomxelliott**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-719

### CODE REVIEW

File: src/test/steps/you-need-to-review-your-application.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  applicant1AlreadyAppliedForHelpPaying: YesOrNo.YES,
+  inTheUk: YesOrNo.NO,
+  certifiedTranslation: YesOrNo.YES,
+  applicant1HelpPayingNeeded: YesOrNo.YES,
+  ceremonyCountry: 'Mozambique',
+  'relationshipDate-day': 31,
+  'relationshipDate-month': 12,
+  'relationshipDate-year': 1999,
+  ceremonyPlace: 'Maputo',
+};
```
Comment (lara-mpt) [inline src/test/steps/you-need-to-review-your-application.ts:19]: caseData might need to be updated after my recent commit - https://github.com/hmcts/nfdiv-frontend/commit/43ceab0275d5cb9725301b524cf4dfa9bcf8c694

I've made it so the application can only be sent for review once all the required fields are filled in

Comment (tomxelliott) [reply src/test/steps/you-need-to-review-your-application.ts:19]: I seem to be getting a Green build right now - have you merged that commit?

Comment (tomxelliott) [reply src/test/steps/you-need-to-review-your-application.ts]: Have updated the PR to use your full case fixture from your PR

**Review by lara-mpt — APPROVED** [review]

File: src/test/functional/features/you-need-to-review-your-application.feature [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    And the page should include "to end your civil partnership"
+
+  Scenario: The page should render husband and divorce content
+    Given I am reviewing an application for divorce created by my husband
+    When I go to '/applicant2/you-need-to-review-your-application'
+    Then the page should include "husband"
+    And the page should include "a divorce application"
+    And the page should include "for divorce"
+
+
```
Comment (lara-mpt) [inline src/test/functional/features/you-need-to-review-your-application.feature:26]:
```suggestion

```

**Pull request merged**